File: 11.py
from sys import stdin
import numpy as np

for line in stdin:
    line = line.split()
    rows, cols = int(line[0]), int(line[1])
    matrix = []
    for i in range(rows):
        matrix.append([int(x) for x in input().split()])
    matrix = np.transpose(np.array(matrix)).tolist()
    for i in range(cols):
        print(' '.join(str(num) for num in matrix[i]))

File: 11.cpp
#include <iostream>
#include <cmath>
using namespace std;

int main()
{
    int rows, cols;
    while (cin >> rows >> cols)
    {
        int arr[rows][cols];
        for (int i = 0; i < rows; i++)
        {
            for (int j = 0; j < cols; j++)
            {
                cin >> arr[i][j];
            }
        }

        // printf("%d %d\n", cols, rows);
        for (int i = 0; i < cols; i++)
            for (int j = 0; j < rows; j++)
            {
                if (j == rows - 1)
                    printf("%d\n", arr[j][i]);
                else
                    printf("%d ", arr[j][i]);
            }
    }
}